projects/showcase/src/app/demo/demo.module.ts

```diff
@@ -71,76 +69,72 @@ import { NgxTerrainSourceComponent } from './examples/ngx-terrain-control.compon
 import { MapTestingHelperDirective } from '../helper/map-testing-helper.directive';
 
 @NgModule({
-  imports: [
-    RouterModule,
-    HttpClientModule,
-    CommonModule,
-    FormsModule,
-    NgxMapLibreGLModule,
-    LayoutModule,
-    MatRadioModule,
-    MatButtonToggleModule,
-    MatButtonModule,
-    MatListModule,
-    MatCardModule,
-    MatProgressSpinnerModule,
-    MatInputModule,
-    MatIconModule,
-    MatSidenavModule,
-    MatPaginatorModule,
-    MatSlideToggleModule,
-  ],
-  providers: [StackblitzEditGuard],
-  declarations: [
-    DemoIndexComponent,
-    MglMapResizeDirective,
-    StackblitzEditComponent,
-    DisplayMapComponent,
-    CustomStyleIdComponent,
-    SetStyleComponent,
-    SatelliteMapComponent,
-    AddImageGeneratedComponent,
-    AddImageComponent,
-    ToggleLayersComponent,
-    ClusterComponent,
-    HeatMapComponent,
-    GeoJSONLineComponent,
-    NgxGeoJSONLineComponent,
-    CustomMarkerIconsComponent,
-    NgxCustomMarkerIconsComponent,
-    LiveUpdateFeatureComponent,
-    LiveUpdateImageSourceComponent,
-    PopupComponent,
-    SetPopupComponent,
-    FullscreenComponent,
-    NavigationComponent,
-    LocateUserComponent,
-    AttributionPositionComponent,
-    NgxScaleControlComponent,
-    NgxCustomControlComponent,
-    InteractiveFalseComponent,
-    LanguageSwitchComponent,
-    CenterOnSymbolComponent,
-    NgxDragAPointComponent,
-    DragAMarkerComponent,
-    HoverStylesComponent,
-    PopupOnClickComponent,
-    ZoomtoLinestringComponent,
-    ClusterHtmlComponent,
-    ClusterPointComponent,
-    Display3dBuildingsComponent,
-    PolygonPopupOnClickComponent,
-    NgxClusterHtmlComponent,
-    ClusterPopupComponent,
-    AddImageMissingGeneratedComponent,
-    CustomAttributionComponent,
-    CustomLocaleComponent,
-    MarkerAlignmentComponent,
-    TerrainMapStyleComponent,
-    TerrainMapComponent,
-    NgxTerrainSourceComponent,
-    MapTestingHelperDirective,
-  ],
-  exports: [DemoIndexComponent],
+    imports: [
+        RouterModule,
+        HttpClientModule,
```

Comment:
Is there a way to remove these modules as well or they are a must due to routing stuff?

Reply:
Probably can be removed as well. Need to have another look at the remaining modules.